pkgs/development/php-packages/relay/default.nix

```diff
@@ -79,16 +74,19 @@ stdenv.mkDerivation (finalAttrs: {
   };
   nativeBuildInputs = lib.optionals (!stdenv.hostPlatform.isDarwin) [ autoPatchelfHook ];
   buildInputs = lib.optionals (!stdenv.hostPlatform.isDarwin) [
+    hiredis
+    libck
     openssl
     zstd
     lz4
   ];
+  internalDeps = [ php.extensions.session ];
   installPhase =
     ''
       runHook preInstall
 
       mkdir -p $out/lib/php/extensions
-      cp relay-pkg.so $out/lib/php/extensions/relay.so
+      cp relay.so $out/lib/php/extensions/relay.so
       chmod +w $out/lib/php/extensions/relay.so
```

Comment:
I think you could replace these 4 lines with one single using the `install` command.